docs/source/get-started/suggested-models.md

```diff
@@ -0,0 +1,187 @@
+# Suggested Models
+
+Lumigator supports any model uploaded to the [Hugging Face Hub](https://huggingface.co/models?pipeline_tag=summarization&sort=trending)
+and trained for *summarization*, provided the model is compatible with the required library versions
+(e.g., Transformers) and runtime dependencies (e.g., vLLM). Practical factors such as compute
+availability and system configurations may also impact the successful use of a model. To ensure a
+seamless experience, [we have extensively tested a few models](https://blog.mozilla.ai/on-model-selection-for-text-summarization/)
+and created an endpoint to easily retrieve them.
+
+In this guide, we assume that you have already [installed Lumigator locally](quickstart), and have a
+running instance. To get a list of suggested models, you can use the following command:
+
+::::{tab-set}
+
+:::{tab-item} cURL
+:sync: tab1
+
+```console
+user@host:~/lumigator$ curl -s http://localhost:8000/api/v1/models/summarization | jq
+{
+  "total": 9,
+  "items": [
+    {
+      "name": "facebook/bart-large-cnn",
+      "uri": "hf://facebook/bart-large-cnn",
+      "description": "BART is a large-sized model fine-tuned on the CNN Daily Mail dataset.",
+      "info": {
+        "parameters_count": "406M",
```

Comment:
we still have some instances of parameters_count rather than parameter_count in this pr, let's make it consistent

Reply:
Fixed. Good catch!